plugins/nodes/src/js/types/IWidthArgs.tsx

```diff
@@ -0,0 +1,8 @@
+// TODO: we need another release which exports this type so we can import
+// import { IWidthArgs } from "@dcos/ui-kit/packages/table/components/Column";
+export interface IWidthArgs {
```

Comment:
Didn't we settle on interfaces without a capital I in the beginning?

Reply:
yep, but they do have it in UI Kit and this is copied from there (see TODO above). we want to change them there -> different Issue.

Reply:
But we could export it without an I here, right?

Reply:
I wanted it to be consitent with UI-Kit, once we changed it there we can change it here I'd say. 🤔

Reply:
We have two systems and as you need to be inconsistent with one of them I would root for the one we decided for in the discussion 🙈 Even if the next release of ui-kit does not have the interfaces renamed we can still do something like `export { IWidthArgs as WidthArgs } from "..."`

Reply:
That was my worst fear :D. I think we should talk with @weblancaster and @Poltergeist if we could have the I removed for everything on the ui-kit. As it will be forever a source of pain.

Reply:
agree, but not in this PR, could you please create a JIRA for that? I already created 20+ JIRAs in the last few days 🙈

Reply:
https://jira.mesosphere.com/browse/DCOS-39139